### PR TITLE
Added button to open gallery picker and used the correct contract to pick an image

### PR DESCRIPTION
**Pull request by ojasvib**
Added button to open gallery picker and used the correct contract to pick an image

### CODE REVIEW

File: app/src/main/java/com/ojasvi/memify/MainActivity.kt [hunk 1]
```diff
@@ -57,27 +61,36 @@ fun MainScreen(modifier: Modifier = Modifier) {
...
     Column(
         horizontalAlignment = Alignment.CenterHorizontally,
         verticalArrangement = Arrangement.Center,
-        modifier = Modifier.fillMaxSize()
-    ){
+        modifier = Modifier.fillMaxSize().padding(20.dp)
+    ) {
+        Button(
+            onClick = { pickMedia.launch(PickVisualMediaRequest(ActivityResultContracts.PickVisualMedia.ImageOnly)) },
+            modifier = Modifier.fillMaxWidth().height(50.dp)
```
Comment (sanskar10100) [inline app/src/main/java/com/ojasvi/memify/MainActivity.kt]: Static heights are bad. Can cut the content off on some devices.

File: app/src/main/java/com/ojasvi/memify/MainActivity.kt [hunk 2]
```diff
@@ -57,27 +61,36 @@ fun MainScreen(modifier: Modifier = Modifier) {
...
+            Text(text = "Open Gallery Picker")
+        }
+
         imageUri?.let { uri ->
-            Image(painter = rememberAsyncImagePainter(uri), contentDescription = null,
-                modifier = Modifier.size(200.dp).clip(CircleShape).border(2.dp, MaterialTheme.colorScheme.onBackground, CircleShape))
+            Image(
+                painter = rememberAsyncImagePainter(uri), contentDescription = null,
+                modifier = Modifier
+                    .size(200.dp)
```
Comment (sanskar10100) [inline app/src/main/java/com/ojasvi/memify/MainActivity.kt]: Try to use fillMaxWidth(0.5f) or something alone the lines. Fixed sizes are bad when you're running the app on different devices.

**Review by sanskar10100 — CHANGES_REQUESTED** [review]

**Review by sanskar10100 — APPROVED** [review]
Merge after resolving both points

File: app/src/main/java/com/ojasvi/memify/MainActivity.kt (hunk 1 above)
Comment (ojasvib) [reply app/src/main/java/com/ojasvi/memify/MainActivity.kt]: Updated to use wrapContentHeight() instead

File: app/src/main/java/com/ojasvi/memify/MainActivity.kt (hunk 2 above)
Comment (ojasvib) [reply app/src/main/java/com/ojasvi/memify/MainActivity.kt]: Updated to use  fillMaxWidth(0.7f) as suggested

**Pull request merged**